docs/core/testing/mstest-analyzers/mstest0007.md

```diff
@@ -0,0 +1,54 @@
+---
+title: "MSTEST0007: [<TestAttribute>] can only be set on methods marked with [TestMethod]"
+description: "Learn about code analysis rule MSTEST0007: Ensure test attributes can only be set on methods marked with TestMethod attribute"
+ms.date: 02/01/2024
+f1_keywords:
+- MSTEST0007
+- UseAttributeOnTestMethodAnalyzer
+helpviewer_keywords:
+- UseAttributeOnTestMethodAnalyzer
+- MSTEST0007
+author: cvpoienaru
+ms.author: codrinpoienaru
+dev_langs:
```

Comment:
Lines 13-15 aren't needed since there are no code examples.

Reply:
Removed, thank you !